views/user_info.erb

```diff
@@ -1,8 +1,8 @@
 <div class="p2 bg-green white clearfix">
   <% if userinfo %>
-    <form action="/auth/saml/logout" method="POST" class="right sm-show">
-      <button type="submit" class="btn p0 h6 white">Log out</button>
-    </form>
+    <div class="right sm-show">
+      <a href="/" class="btn p0 h6 white">Log out</button>
```

Comment:
I am not clear on why going to the root will log someone out? Or is this just to make the link work but not actually function to do the logout stuff?